test/extended/builds/pipeline_jenkins_e2e.go

```diff
@@ -454,7 +452,7 @@ var _ = g.Describe("[Slow]jenkins repos e2e openshift using slow samples pipelin
 				o.Expect(envs[0].Value).To(o.Equal("BAR1"))
 			})
 
-			g.By("Blue-green pipeline")
+			/*g.By("Blue-green pipeline")
```

Comment:
Are we going to revert this?

Reply:
either revert or delete entirely

I'm going to try some runs with both and see how flake this test still is.  Last time I tried it was flaky but that was a couple of months ago.